### DISCUSSION TITLE
Hacks down?

### DISCUSSION THREAD

**Discussion by hm9m** [Prodigy-Hacking/ProdigyMathGameHacking#1295] [(B) Bug Discussion]
It saying my version is invalid and needs update but i reinstall it  and it still says invaild @ArcerionDev @PatheticMustan 
how to fix?

**Comment by afkvido** [comment]
> It saying my version is invalid and needs update but i reinstall it and it still says invaild @ArcerionDev @PatheticMustan
> how to fix?

Are you on 2.1.8?

**Reply by afkvido** [reply]
If so, then you're on the latest version, and you're fine. 


If you don't have PHEx 2.1.8, download it here: https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/releases/tag/2.1.8

###### Latest Version: https://hacks.prodigyhacking.com/version

**Reply by hm9m** [reply]
it does not let me

**Reply by hm9m** [reply]
says no mainifest found

**Reply by afkvido** [reply]
bruh

**Reply by LeoBadeaux** [reply]
> says no mainifest found

Hey, is this still happening?
